packages/agent-toolkit/src/utils/tools/tools-filtering.utils.ts

```diff
@@ -5,7 +5,7 @@ import { toolFactory } from './initializing.utils';
 import { MondayApiToolContext } from '../../core/tools/platform-api-tools/base-monday-api-tool';
 
 export const getFilteredToolInstances = (
-  instanceOptions: { apiClient: ApiClient; apiToken: string; context?: MondayApiToolContext },
+  instanceOptions: { apiClient: ApiClient; devApiClient: ApiClient; apiToken: string; context?: MondayApiToolContext },
```

Comment:
if you use the latest version of the api client package, there is param that allows you to pass version, so I think if a tool expects to be on dev version which is irregular it can be marked somhow or just directly pass the param, passingm multiple client can be problematic lets say we want now 3 different versions...